src/audit-request-template.md

```diff
@@ -0,0 +1,47 @@
+# Audit Request Template
+*Use this template to communicate estimates and get approval for an audit. Please fill out the relevant sections and get approval from the folks listed below. Once you have received approval, you can engage with audit firms on details of the audit and request quotes. A zip request can then be filed to get spend approval.*
+
+# **Overview**
+
+Link to Audit Readiness Document
+
+## **Timeline and key stakeholders:**
+
+## **When?**
+
+Audit Dates:
+
+Planned Release:
+
+## **Who ?**
+
+Auditors:
+
+OP Labs Facilitators:
+
+## **Costs**
+
+Anticipated Number of weeks:
+
+- 
+
+Expected Cost: ???
+
+- 
+
+## **Approved**
+
+Not started  [Karl Floersch](mailto:karl@oplabs.co)
+
+Action item:
+
+- 
+    
+    [](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEgAAABICAYAAABV7bNHAAAA1ElEQVR4Ae3bMQ4BURSFYY2xBuwQ7BIkTGxFRj9Oo9RdkXn5TvL3L19u+2ZmZmZmZhVbpH26pFcaJ9IrndMudb/CWadHGiden1bll9MIzqd79SUd0thY20qga4NA50qgoUGgoRJo/NL/V/N+QIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIEyFeEZyXQpUGgUyXQrkGgTSVQl/qGcG5pnkq3Sn0jOMv0k3Vpm05pmNjfsGPalFyOmZmZmdkbSS9cKbtzhxMAAAAASUVORK5CYII=)
+    
+    Create a formal zip request once this document has been reviewed and approved.
+    
+
+## References
+
+Created from [**Audit Request**](https://www.notion.so/Audit-Request-1a8f153ee1628045b467c262fae21975?pvs=21)
```

Comment:
Same confusion on the references section, maybe we can just delete these?

Reply:
Made the same fix, I don't want to modify the document in Notion until this is merged.